### ISSUE TITLE
Error on installing amber

### ISSUE THREAD

**Issue by dkniffin** [amberframework/homebrew-amber#3]
```
> brew install amber
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
No changes to formulae.

==> Installing amber from amberframework/amber
==> Downloading https://github.com/amberframework/amber/archive/v0.27.0.tar.gz
Already downloaded: /Users/dkniffin/Library/Caches/Homebrew/downloads/87abc9b212e2021edb5680c7bdaa5694e4eb6efeae7993d54eabf481111af140--amber-0.27.0.tar.gz
==> shards install
==> make install PREFIX=/usr/local/Cellar/amber/0.27.0
Last 15 lines from /Users/dkniffin/Library/Logs/Homebrew/amber/02.make:
                ^~~~~~~~~~~~~~~~~~

in src/amber/support/message_encryptor.cr:23: instantiating 'sign_bytes(Slice(UInt8))'

      if Crypto::Subtle.constant_time_compare(sign_bytes(data_iv), signature)
                                              ^~~~~~~~~~

in src/amber/support/message_encryptor.cr:66: no overload matches 'OpenSSL::HMAC.digest' with types Symbol, String, Slice(UInt8)
Overloads are:
 - OpenSSL::HMAC.digest(algorithm : OpenSSL::Algorithm, key, data)

      OpenSSL::HMAC.digest(@digest, @secret, data)
                    ^~~~~~

make: *** [bin/amber] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/amberframework/homebrew-amber/issues

These open issues may also help:
Error on upgrading amber from 0.11.1 to 0.27.0 https://github.com/amberframework/homebrew-amber/issues/2
```

**Comment by lxxxvi** [comment]
@dkniffin I ran into the same issue. It looks like installation doesn't work if you're trying to compile amber 0.27.0 with crystal 0.28.0 ... See #2

**Comment by dkniffin** [comment]
Yep, looks like that may have been the issue. I cloned down the amber repo and built the `stable` branch, and it works.

**Comment by elorest** [comment]
We released a new version of Amber last night with 0.28.0 support.

**Comment by elorest** [comment]
Brew should work now too.
